Code/Framework/AzCore/AzCore/IO/Streamer/Streamer.h

```diff
@@ -171,6 +174,11 @@ namespace AZ::IO
         //! Call to collect statistics from all the components that make up Streamer.
         void CollectStatistics(AZStd::vector<Statistic>& statistics) override;
 
+        //! Tells AZ::IO::Streamer the report the information for the report to the output.
+        FileRequestPtr Report(AZStd::vector<Statistic>& output, IStreamerTypes::ReportType reportType) override;
+        //! Tells AZ::IO::Streamer the report the information for the report to the output.
```

Comment:
Repeat: This should probably say that this tells AZ::IO::Streamer to collect the information for the report.